modules/nf-core/varlociraptor/callvariants/main.nf

```diff
@@ -4,27 +4,24 @@ process VARLOCIRAPTOR_CALLVARIANTS {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/varlociraptor:8.1.1--hc349b7f_0':
-        'biocontainers/varlociraptor:8.1.1--hc349b7f_0' }"
+        'https://depot.galaxyproject.org/singularity/varlociraptor:8.7.1--ha8ac579_0':
+        'biocontainers/varlociraptor:8.7.1--ha8ac579_0' }"
 
     input:
     tuple val(meta), path(normal_vcf), path(tumor_vcf)
     path (scenario)
     val (scenario_sample_name)
 
     output:
-    tuple val(meta), path("*.bcf.gz"), emit: bcf_gz, optional: true
-    tuple val(meta), path("*.vcf.gz"), emit: vcf_gz, optional: true
-    tuple val(meta), path("*.bcf")   , emit: bcf   , optional: true
-    tuple val(meta), path("*.vcf")   , emit: vcf   , optional: true
-    path "versions.yml"              , emit: versions
+    tuple val(meta), path("*.bcf"), emit: bcf
+    path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}.vcf.gz"
+    def args   = task.ext.args   ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}_called"
```

Comment:
I see what you did there !

Reply:
See https://github.com/nf-core/website/pull/2608 for a long discussion on this point.
I thought it had been merged into the guidelines, I've definitely been doing it for ages ;)